CHANGE only export images linked to variation for variations with col… (#13)
plentymarkets/plugin-elastic-export-check-24-de: meta/documents/changelog_en.md

# Release Notes for Elastic Export Check24.de

## v1.2.1 (2019-09-16)

### Changed

> - Instead of only exporting images linked to the variation in all cases, this will now only apply for variations with an color attribute for other variations all images will be exported.

Reviewer: ```suggestion
- Instead of only exporting images linked to the variation, this will now only apply for variations with a color attribute. For other variations, all images will be exported.
```
